### PR TITLE
Add DFC and DFCS enums, remove corresponding constants and move DrawFrameControl to User32 Interop

### PR DESCRIPTION
**Pull request by gpetrou**
## Proposed changes

- Add DFC and DFCS enums.
- Remove DFC and DFCS constants and replace their usages with the above enums.
- Move DrawFrameControl to User32 Interop.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2250)

### CODE REVIEW

File: src/Common/src/Interop/User32/Interop.DFC.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        public enum DFC
```
Comment (hughbe) [inline src/Common/src/Interop/User32/Interop.DFC.cs]: make uint

Comment (gpetrou) [reply src/Common/src/Interop/User32/Interop.DFC.cs]: Done.

File: src/Common/src/Interop/User32/Interop.DFCS.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        public enum DFCS
```
[the remark this answers is not in the document]
Comment (gpetrou) [reply src/Common/src/Interop/User32/Interop.DFCS.cs]: Done.

File: src/System.Windows.Forms/src/System/Windows/Forms/ControlPaint.cs [hunk 3]
```diff
@@ -1445,8 +1441,7 @@ private static void DrawFlatCheckBox(Graphics graphics, Rectangle rectangle, Col
                         IntPtr dc = g2.GetHdc();
                         try
                         {
-                            SafeNativeMethods.DrawFrameControl(new HandleRef(null, dc), ref rcCheck,
-                                                               NativeMethods.DFC_MENU, NativeMethods.DFCS_MENUCHECK);
+                            SafeNativeMethods.DrawFrameControl(new HandleRef(null, dc), ref rcCheck, (int)User32.DFC.MENU, (int)User32.DFCS.MENUCHECK);
```
[the remark this answers is not in the document]
Comment (gpetrou) [reply src/System.Windows.Forms/src/System/Windows/Forms/ControlPaint.cs]: Done.

**Pull request merged**